src/core/utilities.js

```diff
@@ -59,7 +57,7 @@ function extend( a, b, undefOnly ) {
 		if ( hasOwn.call( b, prop ) ) {
 
 			// Avoid "Member not found" error in IE8 caused by messing with window.constructor
-			if ( !( prop === "constructor" && a === window ) ) {
+			if ( !( prop === "constructor" && a === global ) ) {
```

Comment:
The comment above says this is a browser specific (IE8) issue. Sounds like using `window` is correct here.